images/macos/provision/configuration/finalize-vm.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e -o pipefail
 
 # Close all finder windows because they can interfere with UI tests
 osascript -e 'tell application "Finder" to close windows'
 
-# Ignore available updates to prevent system pop-ups
-updateName=$(softwareupdate -l | grep "Title: " | awk -F[:,] '{print $2}' | awk '{$1=$1};1')
-if [ ! -z "$updateName" ]; then
-    sudo softwareupdate --ignore "$updateName"
+if is_Less_BigSur; then
```

Comment:
Should we add source utils.sh to import this function?

Reply:
looks like! Thanks!